README.md

```diff
@@ -67,7 +67,7 @@ or just look for any ".d.ts" files in the package and manually include them with
 
 #### Older versions of TypeScript (2.8 and earlier)
 
-Definitely Typed only tests packages on versions of TypeScript that are less than 2 years old.
+DefinitelyTyped only tests packages on versions of TypeScript that are less than 2 years old.
```

Comment:
this feels really petty, but I actually do care about spelling the product "Definitely Typed" and only having the path be "DefinitelyTyped". And there aren't any international trademarks that I'm aware of, unlike JavaScript and MicroSoft.

Can you make this change in a separate PR and we can discuss there?

Reply:
I can, but FWIW, that's how the spelling has always been in the docs/site for DT: http://definitelytyped.org/guides/pull-request.html

Reply:
I am assuming we dont want this spelling change in this PR.